composer/utils/batch_helpers.py

```diff
@@ -0,0 +1,99 @@
+from typing import Any
+
+from composer.core.types import BatchKeys
+
```

Comment:
This file will need an `__all__` and class-level docstring -- see https://github.com/mosaicml/composer/blob/dev/STYLE_GUIDE.md#62-use-of-__all__ and https://github.com/mosaicml/composer/blob/dev/STYLE_GUIDE.md#4-public-apis?

Reply:
done